Code/GraphMol/Wrap/rdmolfiles.cpp

```diff
@@ -1633,13 +1701,59 @@ BOOST_PYTHON_MODULE(rdmolfiles) {
        - includeMol: include CTAB (Mol) in the output
 
      RETURNS:
-       a Mol object, None on failure.)DOC";
+       the updated PNG data)DOC";
   python::def(
       "MolMetadataToPNGString", addMolToPNGStringHelper,
       (python::arg("mol"), python::arg("png"), python::arg("includePkl") = true,
        python::arg("includeSmiles") = true, python::arg("includeMol") = false),
       docString.c_str());
 
+  docString =
+      R"DOC(Adds metadata to PNG data read from a file.
+
+     ARGUMENTS:
+
+       - metadata: dict with the metadata to be written 
+                   (keys and values should be strings)
+
+       - filename: the PNG filename
+
+     RETURNS:
+       a string with the updated PNG data)DOC";
+  python::def(
+      "AddMetadataToPNGFile", addMetadataToPNGFileHelper,
+      (python::arg("metadata"), python::arg("filename")),
+      docString.c_str());
+
+  docString =
+      R"DOC(Adds metadata to a PNG string.
```

Comment:
Do we care about the distinction between string and bytes?  I assume this function will work on both.

Reply:
yeah, I don't think we care. I'd rather leave the docs as-is rather than confusing things by mentioning bytes